test/dynamo/test_ctx_manager.py

```diff
@@ -1,13 +1,40 @@
 # Owner(s): ["module: dynamo"]
+import contextlib
+import sys
+import traceback
 import unittest
+from contextlib import contextmanager
 
 import torch
 import torch._dynamo.test_case
 import torch._dynamo.testing
 from torch._dynamo.testing import EagerAndRecordGraphs, normalize_gm, same
+from torch._dynamo.utils import counters
 from torch.nn import functional as F
 from torch.testing._internal.common_cuda import PLATFORM_SUPPORTS_FLASH_ATTENTION
-from torch.testing._internal.common_utils import TEST_WITH_ROCM
+from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
+    parametrize,
+    TEST_WITH_ROCM,
+)
+
+
+def _contextmanager(func):
+    # Copied from CPython! Behave exactly as contextlib.contextmanager but do not
+    # wraps func. It is used internally in a few tests to check if the context
+    # manager support in dynamo is correct. We can't use the original implementation
+    # as it can lead to graph breaks in some unrelated cases because it wraps(func)
+    from contextlib import _GeneratorContextManager
+
+    # @wraps(func)
```

Comment:
We fixed the wraps issue, can we uncomment it now? https://github.com/pytorch/pytorch/issues/141514